modules/client-configuration/pages/registration-overview-almalinux.adoc

```diff
@@ -0,0 +1,10 @@
+[[almalinux-registration-overview]]
+= {almalinux} Client Registration
+
+You can register {almalinux} clients to your {productname} Server.
+The method and details varies depending on the operating system of the client.
```

Comment:
```suggestion
The method and details vary depending on the operating system of the client.
```

Reply:
Heh, I copied thies from https://www.uyuni-project.org/uyuni-docs/uyuni/client-configuration/registration-overview-centos.html and it seems all the other OS have the same (for example: https://www.uyuni-project.org/uyuni-docs/uyuni/client-configuration/registration-overview-suse.html)

I will accept your commit here. Can you fix the other clients?